ledger/error/src/main/scala/com/daml/error/ErrorCategory.scala

```diff
@@ -187,7 +187,7 @@ object ErrorCategory {
   object AuthInterceptorInvalidAuthenticationCredentials
       extends ErrorCategoryImpl(
         grpcCode = Some(Code.UNAUTHENTICATED),
-        logLevel = Level.WARN,
+        logLevel = Level.INFO,
```

Comment:
Doing that for all log messages seems potentially a bit too invasive. E.g., a malicious signature or something like that seems like it could reasonably be logged at warning. What would it take to only apply this to the logs if the user does not exist?

Reply:
The alternative is to create a new error code, or use an existing one.

Returning `USER_NOT_FOUND` ([UserManagementServiceErrorGroup.UserNotFound](https://github.com/digital-asset/daml/blob/main/ledger/ledger-api-errors/src/main/scala/com/daml/error/definitions/groups/UserManagementServiceErrorGroup.scala#L92)) instead of `PERMISSION_DENIED` is slightly dangerous. It would be leaking a bit of information, and it would be the only exception where the auth interceptor doesn't return a bare `PERMISSION_DENIED`/`UNAUTHENTICATED`). Since the user check comes after the JWT signature verification, I don't think it would be dangerous.

We could also duplicate [LedgerApiErrors.AuthorizationChecks.PermissionDenied](https://github.com/digital-asset/daml/blob/main/ledger/ledger-api-errors/src/main/scala/com/daml/error/definitions/groups/AuthorizationChecks.scala#L88) and set the log level of the duplicate to INFO. But then we would have to go through all uses of `PermissionDenied` and decide which ones should be logged at INFO and which ones at WARN, which sounds a bit arbitrary.